avocado-core/src/main/scala/org/bdgenomics/avocado/discovery/ReadExplorer.scala

```diff
@@ -78,6 +79,8 @@ class ReadExplorer(referenceObservations: RDD[Observation]) extends Explorer wit
         quals(readPos),
         mapq,
         negativeStrand,
+        firstOfPair,
+        readPos,
```

Comment:
If you want to store just one field, you can store the first of pair as positive numbers and the second of pair as negative indices. This is a micro-optimization, but it is consistent with BQSRs nomenclature, IIRC.